geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionImpl.java

```diff
@@ -826,15 +834,15 @@ public String toString() {
     }
 
     @Override
-    public void messageReceived(DistributionMessage msg) {
+    public void messageReceived(Message msg) {
       membership.processMessage(msg);
 
     }
 
   }
 
   /** this is a fake message class that is used to flush the serial execution queue */
-  static class FlushingMessage extends DistributionMessage {
+  static class FlushingMessage extends AbstractGMSMessage {
```

Comment:
DistributionImpl is part of geode-core. Should it be using AbstractGMSMessage (not part of the geode-membership API).

Reply:
I'll revert that change.  I didn't realize that DistributionImpl was outside of membership